src/utils/errorUtils.ts

```diff
@@ -95,6 +113,25 @@ function getStringMessage(
   return message;
 }
 
+function getErrorKeyForStringMessageGenerator(
+  defaultErrorKey: string,
+  keyOptions: StringMessageGeneratorKeyOptions | undefined
+) {
+  let errorKey: string | undefined = defaultErrorKey;
+
+  if (keyOptions?.shouldHideErrorKey) {
+    errorKey = undefined;
+  } else if (keyOptions?.customKey) {
+    errorKey = keyOptions?.customKey;
+  } else if (keyOptions?.fieldLabelMap?.[defaultErrorKey]) {
+    errorKey = keyOptions.fieldLabelMap[defaultErrorKey];
+  } else if (keyOptions?.shouldCapitalizeErrorKey) {
```

Comment:
Should we make `shouldCapitalizeErrorKey` and `fieldLabelMap` workable together?

It's an edge case but the developer may want to send `{ phone_number: "custom_title" }` and still want to use `shouldCapitalizeErrorKey`

If we're not gonna do this, we should add this info to the docs.

Reply:
Makes sense 👍  I have changed it.